website/docusaurus.config.js

```diff
@@ -139,4 +153,38 @@ module.exports = {
       },
     ],
   ],
+  // Misleading API, probably will be refactored in Docusaurus, but this is
+  // simply a list of <link> tags
```

Comment:
That's more like a note to myself. I have this comment on multiple websites I helped set up now, but I still can't think of a better API. We'll probably have an `htmlTags` API that allows you to inject arbitrary HTML tags, not just `<link>`?